tests/follow.test.js

```diff
@@ -0,0 +1,84 @@
+import chai from 'chai';
+import chaihttp from 'chai-http';
+import server from '../src/index';
+import status from '../src/helpers/constants/status.codes';
+import generateToken from '../src/helpers/tokens/generate.token';
+import env from '../src/configs/environments';
+import errMessage from '../src/helpers/constants/error.messages';
+
+chai.use(chaihttp);
+chai.should();
+
+describe('POST /api/profiles/:username/follow', () => {
+  it('Should return a message on successful follow', (done) => {
+    const token = generateToken({ username: 'daniel' }, env.secret);
+
+    chai
```

Comment:
can a user follow himself????

Reply:
already solved the issue, I added a constraint  to prevent the row fro having the same user